package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "survive-the-monsters",
+  "version": "1.0.0",
+  "description": "mvp",
+  "main": "index.js",
+  "scripts": {
+    "test": "test"
+  },
+  "author": "grezisek",
+  "license": "ISC",
+  "dependencies": {
+    "phaser": "^3.86.0"
+  }
+}
```

Comment:
**nitpick**: Consider adding a newline at the end of the file for consistency and to follow best practices.

Reply:
this is autogenerated by npm so i'll let it be as is